app/models/ability.rb

```diff
@@ -36,8 +36,8 @@ def initialize(user)
     can :manage, Recipe, user: user # if the user is logged in can manage it's own posts
     can :manage, Inventory, user: user # if the user is logged in can manage it's own comments
 
-    return unless user.admin? # additional permissions for administrators
```

Comment:
Could you please keep all permisions for admn? We will add "admin?" method the the model.